libcloud/loadbalancer/drivers/elb.py

```diff
@@ -308,8 +308,7 @@ def ex_create_balancer_listeners(self, name, listeners=None):
             if protocol == 'HTTPS' or protocol == 'SSL':
                 params['Listeners.member.%d.   \
                         SSLCertificateId' % i] = listener[3]
-        else:
-            return False
+        return False
```

Comment:
`for` `else` is definitely confusing. The method is currently broken. The corresponding test is the part of the three test that start with `def text_...` instead of `def test_...`. I think we should ignore this in the PR and fix it in another PR.